lisa/util/shell.py

```diff
@@ -584,7 +584,10 @@ def remove(self, path: PurePath, recursive: bool = False) -> None:
                        (will fail if that's the case and this flag is off)
         """
         assert isinstance(path, Path), f"actual: {type(path)}"
-        path.rmdir()
+        if path.is_dir():
```

Comment:
While you are at it, also properly handle the `recursive` parameter?

Reply:
Done.